js/Gruntfile.js

```diff
@@ -136,6 +136,7 @@ module.exports = function(grunt) {
 		babel: {
 			options: {
 				presets: ['es2015'],
+				plugins: ["array-includes"],
```

Comment:
Hmm, seems like `String.prototype.includes` (and same with array) is already available for our requirements : https://kangax.github.io/compat-table/es6/#test-String.prototype.contains